src/main/java/roboy/memory/nodes/Roboy.java

```diff
@@ -0,0 +1,132 @@
+package roboy.memory.nodes;
+
+import roboy.dialog.Config;
+import roboy.memory.Neo4jMemory;
+import roboy.memory.Neo4jRelationships;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates a MemoryNodeModel and enables dialog states to easily store
+ * and retrieve information about Roboy.
+ */
+public class Roboy {
+    private MemoryNodeModel roboy;
+    Neo4jMemory memory;
+    // Memory is not queried in NOROS mode.
+    private boolean memoryROS;
+
+    /**
+     * Initializer for the Roboy node
+     */
+    public Roboy() {
+        this.roboy = new MemoryNodeModel(true);
+        this.memory = Neo4jMemory.getInstance();
+        this.memoryROS = Config.MEMORY;
+        this.InitializeRoboy("roboy"); // May be "roboy junior"
```

Comment:
then pass it as a parameter to the constructor?

Reply:
Done